README.md

```diff
@@ -29,22 +29,22 @@ The samples are meant to be used with a custom (prototype) engine using the mast
 
 0. Set up the Framework development environment: see [the wiki page](https://github.com/flutter/flutter/wiki/Setting-up-the-Framework-development-environment).
 
-1. In **the framework repo**, switch to the master branch:
+1. Set up the Engine development environment: see [the wiki page](https://github.com/flutter/flutter/wiki/Setting-up-the-Engine-development-environment).
```

Comment:
Since the wiki has moved, consider:

```suggestion
1. Set up the Engine development environment: see [the wiki page](https://github.com/flutter/flutter/blob/master/engine/src/flutter/docs/contributing/Setting-up-the-Engine-development-environment.md).
```